dm/master/openapi.go

```diff
@@ -154,3 +203,36 @@ func sendHTTPErrorResp(ctx echo.Context, code int, message string) error {
 	err := openapi.ErrorWithMessage{ErrorMsg: message, ErrorCode: code}
 	return ctx.JSON(http.StatusBadRequest, err)
 }
+
+func sourceCfgToModel(cfg config.SourceConfig) openapi.Source {
+	// NOTE we don't return security content here, because we don't want to expose it to the user.
```

Comment:
but decrypted password is returned

Reply:
I'm not quite sure if I need to hide the password here, because the API usage scenario is a bit different from ctl
When the user uses ctl, he has a copy of the yaml archive, so the yaml has a record of the password, but if the API is used, the user is uploading the plain text and may don't have raw config backup, so it would be a bit strange if we encrypted it and sent it back🤔

Reply:
I remember scheduler stores plain password, so user has uploaded an encrypted password but this API will decrypt it

Reply:
😂 yes it's, but  there is no way to know if the user upload is encrypted or not (because the scheduler does not store such information)，however, there is no indication in the API documentation that the change field can be encrypted by dmctl,so maybe it's not a problem at now

Reply:
“but if the API is used, the user is uploading the plain text and may don't have raw config backup” 
There is no need to consider this case, the user needs to save the password himself in a secure way, and we should ensure that the received plaintext password is always saved and transmitted in an encrypted way @Ehco1996 @lance6716